ui/pages/lock.tsx

```diff
@@ -365,6 +363,7 @@ export default function Lock() {
                     min={minMaxLockTime.min}
                     max={minMaxLockTime.max}
                     onChange={(e: any) => {
+                      if(e.target.value < minMaxLockTime.min) { return false;}
```

Comment:
```suggestion
                      if (e.target.value < minMaxLockTime.min) {
                        return false;
                      }
```

Reply:
I much prefer this (your) syntax, but we should probably have prettier or something to enforce it? (When i run `npm run lint` the line you changed is not flagged as problematic.